api/v1beta1/clusterclass_types.go

```diff
@@ -826,6 +843,7 @@ type ValidationRule struct {
 	// Example:
 	// "x must be less than max ("+string(self.max)+")"
 	// +optional
+	// +kubebuilder:validation:MaxLength=512
```

Comment:
Given CEL is verbose, a message expression may be fairly long, I'd probably double this, 1024

Reply:
Will do